opentracing-jaxrs2/src/main/java/io/opentracing/contrib/jaxrs2/server/OperationNameProvider.java

```diff
@@ -103,4 +104,39 @@ public static Builder newBuilder() {
       return new Builder();
     }
   }
+
+  /**
+   * As operation name provides the path annotation from both the class and the method e.g:
+   *
+   * resource class annotated with @Path("/foo/") and method annotated with
+   * @Path("bar/{name: \\w+}") produces "/foo/bar/{name}"
+   */
+  class PathAnnotationOperationName implements OperationNameProvider {
```

Comment:
Could you please move this implementation to `WildcardOperationName` and provide a test?

Reply:
I've might been a bit to quick and copied the javadoc. My solution doesn't strip the patterns from the variables, it just uses the plain value of the path annotation.

Do you know a solution for resolving variable names independent from the JAX-RS implementation?